src/mono/mono/metadata/threads.c

```diff
@@ -1092,6 +1096,11 @@ mono_thread_detach_internal (MonoInternalThread *thread)
 
 	MONO_PROFILER_RAISE (thread_exited, (thread->tid));
 
+#ifdef ENABLE_PERFTRACING
+	extern void mono_eventpipe_raise_thread_exited (uint64_t);
+	mono_eventpipe_raise_thread_exited (thread->tid);
+#endif
```

Comment:
As a future TODO, it would be nice if we could piggyback on `MONO_PROFILER_RAISE (thread_exited, ...)` above to get this behavior.  I believe the debugger does something similar, so there is precedent.

Reply:
Was a little unsure of overhead always setting up a profiler since eventpipe might end up enabled, but if the cost is small, we could of course use existing profiler callbacks to track thread_exit.

Reply:
Did switch to profiler callback as part of this PR.